synadm/cli/__init__.py

```diff
@@ -446,7 +453,8 @@ def get_redacted_token_prompt(cli_token):
         "format": click.prompt(
             "Default output format",
             default=output if output else helper.config.get("format", output),
-            type=click.Choice(["yaml", "json", "human", "pprint"])),
+            type=click.Choice([
+                "yaml", "json", "minified", "human", "pprint"])),
```

Comment:
(minor) Appears to be a duplicate of https://github.com/JOJ0/synadm/pull/92/files#diff-80ac2e90b8bc961d0c6426a1304211ae1d7720562d32d2b40e64f97a0d9b55ceR71-R77

Reply:
Sorry I don't understand? Did I commit the same thing twice? I force pushed a couple of times. That the issue?

Reply:
Is there anyghing to do here still, sorry I really didn't get what you mean. Please elaborate! Do you mean we could use the FORMATTERS dict to provide the choices with `click.choice`?
Docs say it needs a list: https://click.palletsprojects.com/en/8.1.x/options/#choice-options
You mean we cast the dict to a list and pass it here? Anyway, please elaborate what you ment! :-) Thanks

Reply:
> Do you mean we could use the FORMATTERS dict to provide the choices with `click.choice`?

that's what I intended to say.

> Docs say it needs a list: https://click.palletsprojects.com/en/8.1.x/options/#choice-options
> You mean we cast the dict to a list and pass it here? Anyway, please elaborate what you ment! :-) Thanks

you can use `dict.keys()` (`dict` being the actual dictionary object) and it will give you a list containing the keys of the dictionary, which you can use to pass to `click.Choice` (if you need the keys but only the first letters, you'll at least need list comprehension)

Reply:
I think I'll decide against my own idea due to some reasons:
1. Every time the program is run, the values has to be computed
2. Code becomes a bit too clever, which is not really great
3. Understanding what the values will be will require jumping around the code more

Reply:
I'm fine with that decision.